### PR TITLE
Docs: Introduce project roadmap

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Specific configuration for apps using GraphQL.
+
+> [ESLint Plugin GraphQL][eslint-plugin-graphql] is great candidate.
+> [GraphQL ESLint plugin][graphql-eslint-plugin]
+
+### Markdown
+
+We should be able lint and format our readmes and other markdown documentation.
+
+> [Remark CLI][remark-cli] is great candidate.
```
Comment (adamkudrna) [inline ROADMAP.md:48]: If linting is all we need, I have quite a good experience with https://github.com/DavidAnson/markdownlint-cli2 (compatible with MDX, to certain extent).

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## New packages to add
+
+### ESLint
+
+We should be able to lint our javascript projects through ESLint and its configuration.
+
+#### Base
+
+Base configuration which should be used everywhere.
```
Comment (adamkudrna) [inline ROADMAP.md:13]: Could you give an example?

**Review by adamkudrna — COMMENTED** [review]
Have you considered using another platform for this kind of information? Roadmap is actually more like (epic) issues rather than documentation that needs to be part of repository. Are you sure that Projects or Wiki up here are not a more convenient choice?

File: ROADMAP.md (hunk 2 above)
Comment (literat) [reply ROADMAP.md:13]: https://github.com/strvcom/code-quality-tools/tree/master/packages/eslint-config-base

Comment (adamkudrna) [reply ROADMAP.md:13]: Oh, I'm sorry, I missed it's all subsections of `### ESLint`… My bad!

**Comment by adamkudrna** [comment]
> Have you considered using another platform for this kind of information? Roadmap is actually more like (epic) issues rather than documentation that needs to be part of repository. Are you sure that Projects or Wiki up here are not a more convenient choice?

E.g. like [MUI Public Roadmap](https://github.com/mui-org/material-ui/projects/25).

**Comment by literat** [comment]
> Have you considered using another platform for this kind of information? Roadmap is actually more like (epic) issues rather than documentation that needs to be part of repository. Are you sure that Projects or Wiki up here are not a more convenient choice?

Well, markdown file was a first simple choice to put my thoughts on paper. But maybe GitHub Project and Issues are far more better for this use case than simple MD file. I'll give it a try and I could return to MD anytime :-)

**Pull request closed without merging**